src/react-components/ory/helpers/node.tsx

```diff
@@ -218,13 +218,24 @@ export const Node = ({
     }
   } else if (isUiNodeAnchorAttributes(node.attributes)) {
     return (
-      <ButtonLink
-        href={node.attributes.href}
-        title={node.attributes.title.text}
-        data-testid={`node/anchor/${node.attributes.id}`}
+      <div
+        style={{
+          display: "inline-flex",
+          flexWrap: "wrap",
+          gap: pxToRem(48),
```

Comment:
why do we need to create such a large gap? and what is the purpose of the div wrapping the buttonlink?

Reply:
>why do we need to create such a large gap?

Good point, copy-pasted from above... removed.

>what is the purpose of the div wrapping the buttonlink?

So that it's centered. We don't use the anchor links in kratos, apart from in my new verification strategy, so I thought that should be fine.